src/main/resources/swagger/api-docs.yaml

```diff
@@ -2997,6 +2997,34 @@ components:
             type: string
       description: ""
     AccessPolicyMembershipV2:
+      required:
+        - actions
+        - memberEmails
+        - roles
+      type: object
+      properties:
+        memberEmails:
+          type: array
+          items:
+            type: string
+        memberPolicies:
+          type: array
+          items:
+            $ref: '#/components/schemas/PolicyInfoResponseBody'
+        actions:
+          type: array
+          items:
+            type: string
+        roles:
+          type: array
+          items:
+            type: string
+        descendantPermissions:
+          type: array
+          items:
+            $ref: '#/components/schemas/AccessPolicyDescendantPermissions'
+      description: ""
+    AccessPolicyMembershipRequest:
       required:
         - actions
         - memberEmails
```

Comment:
@sjkobori `memberEmails` shouldn't be marked as required, correct? You have to supply either the `memberEmails` or the `memberPolicies` (I will be supplying only `memberPolicies`).

Reply:
That should be updated in the swagger doc, I can make a ticket t update that. But it functionally works the way you are describing. Providing either will be sufficient.

Reply:
If you provide an empty list for `memberEmails` and provide `memberPolicies` the endpoint should work correctly. Currently, the documentation aligns with the functionality, as `memberEmails` is not an optional value. It may make sense to change that going forward, but in general, I want to make a pass on all of the collections we are using across our endpoints. I want make sure that we have a consistent way of handling empty collections before making a change here.